amit/index.js

```diff
@@ -0,0 +1,30 @@
+let snackServings = 22;
+let guests = 25;
+let add = snackServings + guests;
+let sub = snackServings - guests;
+let mal = snackServings * guests;
+let div = snackServings / guests;
+let drinks = 10;
+let resultOne = mal / drinks;
+console.log(resultOne);
+
+let monkeys = 15;
+let bananas = 9;
+let div2 = monkeys / bananas;
+
+let monkeyfights = 20;
+let add2 = monkeys + bananas;
+let mal2 = add2 * monkeyfights;
+let resultTwo = mal2;
+console.log(resultTwo);
+
+monkeys = monkeys + 10;
```

Comment:
This is the correct way to increment monkeys and decrement bananas 
console.log(++monkeys);
console.log(--bananas);